sdk/aqueduct/flow.py

```diff
@@ -153,3 +158,28 @@ def describe(self) -> None:
             )
         )
         print(json.dumps(self.list_runs(), sort_keys=False, indent=4))
+
+    def artifact(
+        self, artifact_name: str
+    ) -> Union[TableArtifact, MetricArtifact, CheckArtifact, ParamArtifact, None]:
+        """Gets the Artifact from the flow based on artifact_name"""
+        flow_run = self.latest()
```

Comment:
I think this `artifact()` method should be on the FlowRun object, not the flow. Always fetching from the latest run seems like an unnecessary restriction. A user should be able to fetch any historical run, and realize the artifact from that historical run, using latest data.